tests/integration/test_agent_loop.py

```
@@ -32,6 +32,25 @@ def get_weather(location: str) -> str:
+
+@pytest.mark.parametrize(
+    ("content", "expected"),
+    [
+        (None, False),
+        ("", False),
+        ("It rains in Paris.", False),
+        ("It is 15C in Paris.", True),
+        ("Sunny in London.", True),
+    ],
+)
+def test_mentions_tool_result(content: str | None, expected: bool) -> None:
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)